system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts

```diff
@@ -15,13 +15,14 @@ ALERT OpenstackCinderVolumeStuck
   }
 
 ALERT OpenstackCinderVolumeAttach
-  IF max(openstack_stuck_volumes_max_duration_gauge{status="attaching"}) by (host) > 15
+  IF max(openstack_stuck_volumes_max_duration_gauge{status="attaching"}) by (host,check) > 15
```

Comment:
Sure, we just want `max(..) > 15`? Looking in staging, the value for that is 800551.598317

Reply:
yes only for `detaching` state its 801091.603697. We working on detaching issue, but the PR raised is for Attaching, where the value for the below query is 0.

max(openstack_stuck_volumes_max_duration_gauge{status="attaching"}) by (host,check) > 15

Reply:
Confusing. Then this LGTM.